app/utilities/simplejson.py

```diff
@@ -0,0 +1,13 @@
+import simplejson as json
+
+
+def load_json(file, **kwargs):
+    return json.load(file, **kwargs)
+
+
+def loads_json(data, **kwargs):
+    return json.loads(data, use_decimal=True, **kwargs)
+
+
+def dumps_json(data, **kwargs):
```

Comment:
I think the naming would read better as `json_load`, `json_loads` and `json_dumps`.

Reply:
I took it from your suggestion in the other PR